doc/particles.md

```diff
@@ -246,14 +253,15 @@ await Flame.images.loadAll(const [
 game.add(
     ParticleComponent(
         particle: ImageParticle(
-          size: const Size.square(24),
+          size: Vector2.all(24),
           image: Flame.images.loadedFiles['image.png'],
```

Comment:
We probably should load the image into a variable before using and then just pass it here. This way by accessing the loaded files seems a little hack and I would avoid something like that on our docs. Wdyt?

Reply:
Agreed :-) will follow up on this